Backend/src/test/java/ca/mcgill/cooperator/service/CooperatorServiceStudentTests.java

```diff
@@ -205,7 +205,7 @@ public void testUpdateStudent1() {
         try {
             Student s = studentService.getStudentByStudentId(studentID);
             studentService.updateStudent(
-                    s, firstName, lastName, email, studentIDNew, coops, notifications);
+                    s, firstName, lastName, email, studentIDNew, coops, notifications, null);
```

Comment:
Might want to update the tests at some point so that we test updating students with reports

Reply:
i did this in CooperatorServiceStudentReportTests